getOption_template

```diff
@@ -0,0 +1,51 @@
+function opt = getOption()
```

Comment:
I created an empty template for the getOptions so it would always be there.
but see if you like the idea or not. Otherwise remove it. I just kept the old empty file as a backup